DataManagementSystem/Client/test/Test_Client.py

```diff
@@ -0,0 +1,150 @@
+import unittest
+from mock import Mock
+
+
+
+from DIRAC.DataManagementSystem.Client.ConsistencyChecks import ConsistencyChecks
+
+
+
+
+class UtilitiesTestCase( unittest.TestCase ):
+
+  def setUp( self ):
+    self.bkClientMock = Mock()
+    self.bkClientMock.getFileDescendants.return_value = {'OK': True,
+                                                         'Value': {'Failed': [],
+                                                                   'NotProcessed': [],
+                                                                   'Successful': {'aa.raw': ['bb.raw', 'bb.log']},
```

Comment:
Since you are doing some os.path operation, I would recommend to avoid path like 'aa.raw', and give real path like '/lhcb/something/aa.raw'